### PR TITLE
update dependency claim to send a combo email if both 686c and 674 ar…

### PR DESCRIPTION
**Pull request by evansmith**
…e present

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- *This work is behind a feature toggle (flipper): NO*
- Updated the send_failure_email method for a dependency_claim to send a combo email of 686c-674 instead of two individual emails.
- The solution is to check for 686 and 674 and decide which template to use.
- VA Benefits and Claims
- 
## Related issue(s)

[- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95830)
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [x] *New code is covered by unit tests*
- Previously we send two emails if it was a 686c-674, now we will send one.
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by mjknight50 — APPROVED** [review]
GTG

File: app/models/saved_claim/dependency_claim.rb [hunk 1]
```diff
@@ -140,22 +140,26 @@ def to_pdf(form_id: FORM)
...
-          {
-            'first_name' => parsed_form.dig('veteran_information', 'full_name', 'first')&.upcase.presence,
-            'date_submitted' => Time.zone.today.strftime('%B %d, %Y'),
-            'confirmation_number' => confirmation_number
-          }
-        )
-      end
+    # if the claim is both a 686c and a 674, send a combination email.
+    # otherwise, check to see which individual type it is and send the corresponding email.
+    template_id = if submittable_686? && submittable_674?
```
Comment (micahaspyr) [inline app/models/saved_claim/dependency_claim.rb:145]: Put in a final else statement when setting the template_id that logs an error or warning message stating that none of those method calls were valid, then set template_id to nil

**Review by micahaspyr — CHANGES_REQUESTED** [review]
Just the one logging change please

**Review by micahaspyr — APPROVED** [review]
LGTM

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**